R/pretty.R

```diff
@@ -35,6 +35,36 @@ renv_pretty_print <- function(values,
   writef(text)
 }
 
+renv_pretty_bullets <- function(values,
```

Comment:
I'll probably pull this out into a separate PR. It's possible we can use it everywhere that `renv_pretty_print()` is currently used.

Reply:
It would also be a good opportunity to revisit the `wrap` argument for `renv_pretty_print`; I think we almost always use `wrap = FALSE` so that is probably the better default.